Documentation/KnownIssues/README.md

```diff
@@ -0,0 +1,9 @@
+# .NET Framework Known Issues
+
+The following links provide the known issues, about the various .NET Framework releases, as they existed at the time of release. They describe the underlying issue and how to resolve them. 
```

Comment:
better: The following links provide the known issues about the various .NET Framework versions at the time of their release.